examples/run_heavy_hex_sampling.py

```diff
@@ -0,0 +1,75 @@
+"""Example building a large heavy-hex circuit with 3 repeated layers of random gates,
+generating the tensor network representation using ITN and then sampling bitstrings"""
+
+import random
+from datetime import datetime
+
+import numpy as np
+from juliacall import Main as jl
+from qiskit import transpile, QuantumCircuit
+from qiskit_ibm_runtime.fake_provider import FakeSherbrooke
+
+from itensornetworks_qiskit.graph import extract_cx_gates
+from itensornetworks_qiskit.ibm_device_map import ibm_qubit_layout
+from itensornetworks_qiskit.sample import itn_samples_to_counts_dict
+from itensornetworks_qiskit.utils import qiskit_circ_to_itn_circ_2d
+
+jl.seval("using ITensorNetworksQiskit")
+
+# Any Julia functions from outside our package should be added here
+jl.seval("using ITensorNetworks: siteinds")
+
+# Note here we use a real device graph, which is subtly different from CouplingMap().from_heavy_hex
+# used in other examples. Any connectivity can be used if a map can be provided between qubit
+# indices and a 2D coordinate grid.
+backend = FakeSherbrooke()
+n_qubits = backend.num_qubits
+cmap = backend.coupling_map
+print(f"Created heavy-hex graph with {cmap.size()} qubits")
+
+graph = backend.coupling_map.get_edges()
+
+# Remove duplicates with opposite direction
+graph = [list(s) for s in set([frozenset(item) for item in graph])]
```

Comment:
```suggestion
graph = [list(t) for t in {tuple(sorted(item)) for item in graph}]
```

I haven't really used `frozenset` much in the past, maybe that is best practice in Python... but maybe this could be simplified? I see this is replicated across all our examples though so maybe this is just my lack of Python knowledge